src/main/java/org/terasology/launcher/util/GuiUtils.java

```diff
@@ -23,10 +23,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.awt.Desktop;
-import java.awt.EventQueue;
+import java.awt.*;
```

Comment:
Did this commit from my second computer which had different settings for imports 😠 So this wasn't an intended change